Source/WebKit/webpushd/WebPushDaemon.mm

```diff
@@ -558,6 +573,18 @@ void handleWebPushDMessage(ClientConnection* connection, Span<const uint8_t> enc
 #endif
 }
 
+void Daemon::getOriginsWithPushSubscriptions(ClientConnection* connection, CompletionHandler<void(const Vector<String>&)>&& callback)
+{
+    runAfterStartingPushService([this, bundleIdentifier = connection->hostAppCodeSigningIdentifier(), callback = WTFMove(callback)]() mutable {
+        if (!m_pushService)
+            return callback({ });
+
+        m_pushService->getOriginsWithPushSubscriptions(bundleIdentifier, [callback = WTFMove(callback)](auto&& origins) mutable {
```

Comment:
I don't see why you wrapped callback in a lambda.

Reply:
It's because one of the lambdas is of type `CompletionHandler<void(const Vector<String>&)>` and the other is of type `CompletionHandler<void(Vector<String>&&)>`.

Reply:
Ok, I fixed the `REPLY` macro to work with movable types, so this is cleaned up now.